packages/client/package.json

```diff
@@ -30,25 +30,25 @@
     "MongoDB"
   ],
   "main": "default.js",
-  "browser": "index-browser.js",
   "types": "default.d.ts",
+  "browser": "index-browser.js",
   "exports": {
     "./package.json": "./package.json",
     ".": {
       "require": {
         "types": "./default.d.ts",
         "node": "./default.js",
-        "edge-light": "./wasm.js",
-        "workerd": "./wasm.js",
-        "worker": "./wasm.js",
+        "edge-light": "./default.js",
+        "workerd": "./default.js",
+        "worker": "./default.js",
```

Comment:
While these are redundant, they are actually needed. If `wrangler` does not see its key, it will default to `browser` as a consequence. We will be able to simplify this before GA though, comments below.